adapters/iqx/iqx.go

```diff
@@ -159,3 +149,16 @@ func prepareBidResponse(seats []openrtb2.SeatBid) (*adapters.BidderResponse, []e
 
 	return bidResponse, errs
 }
+
+func getMediaTypeForBid(bid openrtb2.Bid) (openrtb_ext.BidType, error) {
+	switch bid.MType {
+	case openrtb2.MarkupBanner:
+		return openrtb_ext.BidTypeBanner, nil
+	case openrtb2.MarkupVideo:
+		return openrtb_ext.BidTypeVideo, nil
+	case openrtb2.MarkupNative:
+		return openrtb_ext.BidTypeNative, nil
+	default:
+		return "", fmt.Errorf("failed to parse bid mType for impression \"%s\"", bid.ImpID)
```

Comment:
if `bid.ImpID` is 2 then this will result in  `"failed to parse bid mType for impression \"2\"`.  From this error message it is unclear whether  2 is mType value or index from imp array. How about refactoring error message to - `failed to parse bid mtype for impression id \2\`

Reply:
@onkarvhanumante done